release-please-config.json

```diff
@@ -9,8 +9,7 @@
       "release-type": "simple",
       "extra-files": [
         "action.yml",
-        "api/gradle.properties",
-        "README.md"
```

Comment:
We should continue to update the README automatically based on my other suggested change